documentation/cli/change_log.rst

```diff
@@ -4,6 +4,8 @@
 FlexMeasures CLI Changelog
 **********************
 
+* Add options to ``flexmeasures add beliefs`` to 1) read CSV data with timezone naive datetimes (use ``--timezone``to localize the data), 2) read CSV data with datetime/timedelta units (use ``--unit datetime`` or ``--unit timedelta`, 3) remove rows with NaN values, and 4) add filter read-in data by matching values in specific columns (use ``--filter-column`` and ``--filter-value`` together).
```

Comment:
The new unit feature (timedelta/datetime) is not explained in the click.option (for --unit). It's important there and should be featured.

And it's not clear to me/the user at this point - it's for changing datetime/timedelta values in the event_value column into seconds since 1970 / seconds?

What is the use case?